man/systemd.xml

```diff
@@ -194,7 +194,7 @@
     <orderedlist>
       <listitem><para>It is in an active, activating, deactivating or failed state (i.e. in any unit state except for <literal>inactive</literal>)</para></listitem>
       <listitem><para>It has a job queued for it</para></listitem>
-      <listitem><para>It is a dependency of some sort of at least one other unit that is loaded into memory</para></listitem>
+      <listitem><para>It is a dependency, of some sort, of at least one other unit that is loaded into memory</para></listitem>
```

Comment:
I would prefer to remove it as you suggested or change `of some sort` to `(either implicitly or explicitly)`.

Reply:
Gonna opt for brevity since it's a list and just drop the modifier.